cmd/power/power.go

```diff
@@ -19,6 +21,53 @@ const (
 	DOCKER_CGROUPS_FILE = "/proc/self/cgroup"
 )
 
+func PowerMain() {
+	app := cli.NewApp()
+
+	app.Name = os.Args[0]
+	app.Version = config.VERSION
+	app.Author = "Rancher Labs, Inc."
+	app.Email = "sid@rancher.com"
+	app.EnableBashCompletion = true
+	app.Action = power
+	app.HideHelp = true
+	app.Flags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "f",
+			Usage: "Does not invoke shutdown, and instead performs the actual action",
+		},
+		cli.BoolFlag{
+			Name:  "n",
```

Comment:
`-n` means that the power function should not call `sync()` itself.  Whether the Linux kernel does sync internally is irrelevant.  If `-n` is passed we need to not call `sync()`.